Francisco_Trujillo/Assignments/DrawStar/draw_star.py

```diff
@@ -0,0 +1,16 @@
+
+
+
+def draw_star(arr):
+    for element in arr:
+        if type(element) is int:
+            print '*' * element
+        else:
+            print element[:1].lower() * len(element)
```

Comment:
Cool. You can also use element[0], e.g. without a range, since you know for a fact you're only ever interested in the first element here.